src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,53 @@
+.nav {
+  &__list {
+    display: flex;
+    margin: 0;
+    padding: 0;
+    list-style: none;
+  }
+
+  &__item {
+    margin: 0 10px;
+  }
+
+  &__item:first-child {
+    margin-left: 0;
+  }
+
+  &__item:last-child {
+    margin-right: 0;
+  }
+
+  &__link {
+    position: relative;
+    display: flex;
+    align-items: center;
+    height: 60px;
+    font-family: Roboto, sans-serif;
+    font-size: 12px;
+    font-weight: 500;
+    text-decoration: none;
+    text-transform: uppercase;
+    color: $normal-color;
```

Comment:
Ensure that the `$normal-color` variable is defined in your SCSS files. It is used for styling the navigation link color and is defined as `$normal-color: #060b35;` in `src/styles/utils/variables.scss`.